src/com/ableton/VirtualEnv.groovy

```diff
@@ -48,12 +48,13 @@ class VirtualEnv implements Serializable {
       script.error "pyenv root path '${pyenvRoot}' does not exist"
     }
 
-    List activateCommands = [
-      "export PYENV_ROOT=${pyenvRoot}",
-      'export PATH=\$PYENV_ROOT/bin:\$PATH',
-      'eval "\$(pyenv init --path)"',
-      'eval "\$(pyenv init -)"',
-    ]
+    List activateCommands = ['export PATH=\$PYENV_ROOT/bin:\$PATH']
+    if (!script.env.PYENV_ROOT) {
+      // Only export PYENV_ROOT if it isn't part of the Jenkins environment
+      activateCommands.add("export PYENV_ROOT=${pyenvRoot}")
```

Comment:
In a4881678645f5b8c3d2c2019cae84850eea7dcdf, please don't put `export PATH=...` before `export PYENV_ROOT=...`

Reply:
I'll take care of this when I'm back from holidays. Also, I merged https://github.com/Ableton/python-pipeline-utils/pull/95 first, and I will rework this PR slightly to take advantage of that.